NAM/convnet.cpp

```diff
@@ -99,7 +99,7 @@ convnet::ConvNet::ConvNet(const int channels, const std::vector<int>& dilations,
 {
 }
 
-convnet::ConvNet::ConvNet(const double loudness, const int channels, const std::vector<int>& dilations,
+convnet::ConvNet::ConvNet(const NAM_SAMPLE loudness, const int channels, const std::vector<int>& dilations,
```

Comment:
Nit: I don't like `loudness` being of "sample" type--it's a parameter of the model.

But I get that it mainly serves to do arithmetic _with_ samples, so I can live with it.